docs/modules/ROOT/pages/advanced-how-tos/building-olm.adoc

```diff
@@ -0,0 +1,27 @@
+= Building an Operator in {ProductName}
+
+If you are developing an application that aligns with the link:https://operatorframework.io/[Operator Framework], and managing it with link:https://olm.operatorframework.io/docs/[Operator Lifecycle Manager (OLM)], you can build your Operator in {ProductName}. The first procedure in this document explains how to use {ProductName} to build an Operator in the most basic sense--building its controller and bundle images. The second procedure is optional but recommended. It explains how you can automatically update the image references that should be present in your bundle.
+
+
+== Building the operator and the bundle
+
+[NOTE] 
+====
+This procedure assumes that the source code for your Operator's controller and bundle, including the Dockerfiles, are in the same git repository, per OLM convention.
+====
+
+.Procedure
+
+. In the {ProductName} UI,  xref:../how-tos/creating.adoc[create a new application] for your Operator in {ProductName}.
+. In your new Operator application, xref:../how-tos/creating.adoc[add a new component] for your controller. Be sure to specify the correct path to the operator's Dockerfile within its git repository.
```

Comment:
I also don't like the use of "controller" here. An operator image could have several controllers to manage all the operands they need to manage. The difference between a bundle and an operator image is that the first one includes the second one and all the manifests needed to deploy it. Here we are building the operator image, which can have several controllers inside